includes/object-cache.php

```diff
@@ -837,8 +836,8 @@ protected function connectUsingHHVM( $parameters ) {
      */
     public function fetch_info() {
         $options = method_exists( $this->redis, 'getOptions' )
-            ? $this->redis->getOptions()
-            : new stdClass();
+        ? $this->redis->getOptions()
+        : new stdClass();
```

Comment:
There are a couple of places where the indentation is removed, can we turn that rule off and undo it?

Reply:
Seems like this happens during `phpcbf`. Need to keep an eye out for this. There is no particular rule for this as far as I can see.